Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1554,6 +1554,7 @@ def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=Fals
        :rtype: ``str``
     """
     # Turning the urls in the table to clickable
+    headers = headers[:]
```

Comment:
why is that between a code section and its related comment?
please move to a more suitable place.